events/wmt23.md

```diff
@@ -61,6 +61,47 @@ It was organised by [WMT](/wmt).
 
 In 2022, the *News* machine translation task was renamed the *General* machine translation task.
 
+## Scientific papers
+
+### Topics
+
+- Machine translation models (neural, statistical etc. )
+- Analysis of neural models
+- Using comparable corpora
+- Selection and preparation of data
+- Semi-supervised and unsupervised learning for machine translation, transfer learning
+- Multilingual machine translation
+- Incorporating linguistic information into machine translation
+- Machine translation inference
+- Manual and automatic methods for evaluating machine translation
+- Quality estimation
+
+### Research Papers
+
+> Research papers should describe original research corresponding to the categories listed above. Research papers that have been or will be submitted to other meetings or publications must indicate this at submission time, and must be withdrawn from the other venues if accepted and published at WMT 2022.
+
+> We will not accept for publication papers that overlap significantly in content or results with papers that have been or will be published elsewhere. It is acceptable to submit work that has been made available as a technical report (or similar, e.g. in arXiv) without citing it.
+
+> For the research track, papers should be anonymised, be between 6 and 10 pages in length (excluding references) and may include supplementary material.
+
+
+### System Papers
+
+> System papers must describe one or more shared task submissions. System paper submissions that we cannot link to a shared task submission will be rejected without review. System papers can overlap with other published work, and do not have to follow the double submission policy. There is no maximum length for system papers, but normally a short paper (4-6 pages) is appropriate. System papers should not be anonymised.
+
+
+## Poster format
+
+- System description papers will be presentated as posters.
```

Comment:
Do we even need this?

It's not evergreen because of the verb tense, so if we keep it, I think we should just use a quote, or make it past tense.